Lombiq.Tests.UI/Extensions/StringExtensions.cs

```diff
@@ -64,4 +64,10 @@ public static class StringExtensions
             .Join("_", text.Split(_invalidPathCharacters.Value))
             .Replace('.', '_')
             .Replace(' ', '-');
+
+    // Concatenates an array of strings, using the specified separator between each member. Empty or null strings are
+    // filtered out.
+    public static string JoinNotEmptySafe(this string[] strings, string separator = "") =>
```

Comment:
I don't really see why "safe" would be necessary.

```suggestion
    public static string JoinNotEmpty(this string[] strings, string separator = "") =>
```

Reply:
It's "safe" because of `null` checking of items, but it can be `JoinNotNullOrEmpty`.

Reply:
That's better, let's do that.